workspace.dsl

```diff
@@ -0,0 +1,167 @@
+				tags "InternalCollaborationSystem"
+
+				group "Web" {
+
+    				containerBoWeb = container "Back-Office Web" "Back-office website" "Node.js, Azure Web App" {
+    					tags "Microsoft Azure - App Services - Web"
+    				}
+
+    				containerBoRedis = container "Redis" "Distributed session storage"
+    				containerBoAzureActiveDirectory = container "Entra ID" "Stores login details and assigned user groups"
+    				containerBoFileStorage = container "File storage" "Stores all files related to appeals"
+
+				}
+
+				group "API" {
+    				containerBoApi = container "Back-Office API" "" "Node.js, Azure Web App" {
+    					tags "Microsoft Azure - App Services - API"
+    					componentBoStateMachine = component "State machine" "Handles the appeal lifecycle and state transitions"
+    				}
+
+    				containerBoAzureSql = container "Database" "Data store for appeals" "Azure SQL"
+				}
+
+				group "Azure functions" {
+				    containerFunctionAppCasedata = container "Casedata import" "Consumes appellant and LPA submissions from Service Bus" "Function App, JavaScript"{
+						tags "Microsoft Azure - Function Apps", "FunctionApp"
+					}
+					containerFunctionAppDocs = container "Document processing" "Copies blobs from Front-Office, processes virus scanning" "Function App, JavaScript"{
+						tags "Microsoft Azure - Function Apps", "FunctionApp"
+					}
+				}
+			}
+
+			systemIntegration = softwareSystem "Integration Layer" "Event-driven message exchange between applications" {
+				tags "Microsoft Azure - Azure ServiceBus Namespace" "InternalCollaborationSystem"
+
+				containerBoServiceBus = container "Service Bus" "Messaging platform for data exchange" "Azure ServiceBus Namespace" {
+					tags "Microsoft Azure - Azure ServiceBus Namespace"
+
+					group "Commands" {
+						componentCaseCmdTopic = component "Service Bus Appeal submission topic" "Appeal submission commands"
+						componentLpaqCmdTopic = component "Service Bus LPA submission topic" "LPA submission commands"
+					}
+					group "Events" {
+						componentCaseMessageTopic = component "Service Bus Appeal topic" "Appeal messages"
+						componentDocumentMessageTopic = component "Service Bus Document topic" "Appeal document messages"
+						componentServiceUserMessageTopic = component "Service Bus Service User topic" "Service user messages"
+						componentEventMessageTopic = component "Service Bus Event topic" "Event message"
+					}
+				}
+
+				containerODW = container "Operational Data Warehouse (ODW)" "Holds all Planning Inspectorate data so that it can be used for internal purposes" {
+				  tags "Microsoft Azure - Azure Synapse Analytics" "InternalCollaborationSystem"
+				}
+
+				containerEventGrid = container "Azure Event Grid", "Reports virus scan results for uploaded blobs" {
+					tags "Microsoft Azure - Azure Event Grid" "InternalCollaborationSystem"
+				}
+			}
+
+			systemGovUk = softwareSystem "GOV Notify" {
```

Comment:
This needs to move outside the `group "Planning Inspectorate" {` area, which is lines 10 to 129 as it makes this component incorrectly appear within the Planning Inspectorate in the system landscape view